proto/evmos/epochs/genesis.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+package evmos.epochs.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/tharsis/evmos/x/epochs/types";
+
+message EpochInfo {
```

Comment:
@ramacarlucho Why not just `Epoch` instead of `EpochInfo`?

Reply:
Also, Can you add description comments for each field?

Reply:
Sure, ill add comments.